docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -29,11 +29,7 @@ children:
       url: todo-tutorial-scaffolding.html
       output: 'web, pdf'
 
-    - title: 'Adding legacy juggler'
```

Comment:
so we're removing this from the sidebar?

Reply:
Yep. The page is about adding `RepositoryMixin` to the Application but apps generated with `lb4` now have that as part of the template. I guess the page itself should be deleted. I can do that in a follow up PR.